USAGEGUIDE.md

```diff
@@ -5,80 +5,79 @@
-- [Deploying 1Password Connect to Kubernetes](#deploying-1password-connect-to-kubernetes)
-- [Kubernetes Operator Deployment](#kubernetes-operator-deployment)
-- [Usage](#usage)
-- [Configuring Automatic Rolling Restarts of Deployments](#configuring-automatic-rolling-restarts-of-deployments)
-- [Development](#development)
+1. [Prerequisites](#prerequisites)
+2. [Configuration Options](#configuration-options)
+3. [Use Kubernetes Operator with Service Account](#use-kubernetes-operator-with-service-account)
+    - [Create a Service Account](#1-create-a-service-account)
+    - [Create a Kubernetes secret](#2-create-a-kubernetes-secret-for-the-service-account)
+    - [Deploy the Operator](#3-deploy-the-operator)
+4. [Use Kubernetes Operator with Connect](#use-kubernetes-operator-with-connect)
+    - [Deploy with Helm](#1-deploy-with-helm)
+    - [Deploy manually](#2-deploy-manually)
+5. [Logging level](#logging-level)
+6. [Usage examples](#usage-examples)
+7. [How 1Password Items Map to Kubernetes Secrets](#how-1password-items-map-to-kubernetes-secrets)
+8. [Configuring Automatic Rolling Restarts of Deployments](#configuring-automatic-rolling-restarts-of-deployments)
+9. [Development](#development)
+
+---
 
 ## Prerequisites
 
 - [1Password Command Line Tool Installed](https://1password.com/downloads/command-line/)
 - [`kubectl` installed](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 - [`docker` installed](https://docs.docker.com/get-docker/)
-- [A `1password-credentials.json` file generated and a 1Password Connect API Token issued for the K8s Operator integration](https://developer.1password.com/docs/connect/get-started/#step-1-set-up-a-secrets-automation-workflow)
-
-## Deploying 1Password Connect to Kubernetes
-
-If 1Password Connect is already running, you can skip this step.
-
-There are options to deploy 1Password Connect:
-
-- [Deploy with Helm](#deploy-with-helm)
-- [Deploy using the Connect Operator](#deploy-using-the-connect-operator)
-
-### Deploy with Helm
-
-The 1Password Connect Helm Chart helps to simplify the deployment of 1Password Connect and the 1Password Connect Kubernetes Operator to Kubernetes.
-
-[The 1Password Connect Helm Chart can be found here.](https://github.com/1Password/connect-helm-charts)
 
-### Deploy using the Connect Operator
+---
 
-This guide will provide a quickstart option for deploying a default configuration of 1Password Connect via starting the deploying the 1Password Connect Operator, however, it is recommended that you instead deploy your own manifest file if customization of the 1Password Connect deployment is desired.
+## Configuration options
+There are 2 ways 1Password Operator can talk to 1Password servers:
+- **Connect**: It uses the 1Password Connect API to access items in 1Password.
+- **Service Account**: It uses [1Password SDK](https://developer.1password.com/docs/sdks/) and [Service Account](https://developer.1password.com/docs/service-accounts) to access items in 1Password.
```

Comment:
```suggestion
- **Service Account**: It uses the [1Password SDK](https://developer.1password.com/docs/sdks/) and [Service Accounts](https://developer.1password.com/docs/service-accounts) to access items in 1Password.
```

That being said, we could leave out "Service Accounts" altogether since they are not relevant for the end user of the integration.

Reply:
And I'd call them 1Password Service Accounts just to make it clear that they are not Kubernetes Service Accounts